services/order.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from django.db import transaction
+
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: Optional[datetime] = None,
+) -> Order:
+    user = get_user_model().objects.get(username=username)
```

Comment:
The `get()` method is used without handling the `DoesNotExist` exception. Consider using `get_object_or_404()` or wrapping this call in a try-except block to handle the exception and avoid runtime errors.